_includes/configuration/resize-img-proportional.md

```diff
@@ -0,0 +1,23 @@
+## resize_img_proportional
+
+When a user resizes an image in the editor, this option controls whether image is resized proportionally or freely. If set to:
+
+- `true` --- When users resize an image, the image will be resized proportionally (both dimensions of the image are resized by the same percentage of length).
```

Comment:
we should mention, with the true setting, holding down shift while resizing will change the behaviour to free-form
and when set to false, holding shift while resizing will get proportional behaviours

Reply:
yeah this exactly

Reply:
Updated with shift behaviour for both true and false options